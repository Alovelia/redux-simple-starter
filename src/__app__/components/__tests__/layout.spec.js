import { shallow } from 'enzyme';
import React from 'react';
import Layout from '../layout';

describe('<Layout />', () => {
  test('when initializing, the expected output is rendered', () => {
    expect(shallow(<Layout />)).toMatchSnapshot();
  });
});
