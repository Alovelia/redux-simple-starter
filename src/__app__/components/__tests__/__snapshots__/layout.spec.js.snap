// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> when initializing, the expected output is rendered 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Layout
    title="_"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defaultTitle="..."
          defer={true}
          encodeSpecialCharacters={true}
          meta={
            Array [
              Object {
                "content": "...",
                "name": "description",
              },
            ]
          }
          titleTemplate="%s"
        />,
        <InjectIntl(TitleComponent)
          id="_"
        />,
        <ErrorBoundary
          FallbackComponent={[Function]}
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "defaultTitle": "...",
          "defer": true,
          "encodeSpecialCharacters": true,
          "meta": Array [
            Object {
              "content": "...",
              "name": "description",
            },
          ],
          "titleTemplate": "%s",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "id": "_",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "FallbackComponent": [Function],
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defaultTitle="..."
            defer={true}
            encodeSpecialCharacters={true}
            meta={
              Array [
                Object {
                  "content": "...",
                  "name": "description",
                },
              ]
            }
            titleTemplate="%s"
          />,
          <InjectIntl(TitleComponent)
            id="_"
          />,
          <ErrorBoundary
            FallbackComponent={[Function]}
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "defaultTitle": "...",
            "defer": true,
            "encodeSpecialCharacters": true,
            "meta": Array [
              Object {
                "content": "...",
                "name": "description",
              },
            ],
            "titleTemplate": "%s",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "id": "_",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "FallbackComponent": [Function],
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
